code/datums/brain_damage/mild.dm

```diff
@@ -0,0 +1,231 @@
+//Mild traumas are the most common; they are generally minor annoyances.
+//They can be cured with mannitol and patience, although brain surgery still works.
+//Most of the old brain damage effects have been transferred to the dumbness trauma.
+
+/datum/brain_trauma/mild
+
+/datum/brain_trauma/mild/hallucinations
+	name = "Hallucinations"
+	desc = "Patient suffers constant hallucinations."
+	scan_desc = "schizophrenia"
+	gain_text = "<span class='warning'>You feel your grip on reality slipping...</span>"
+	lose_text = "<span class='notice'>You feel more grounded.</span>"
+
+/datum/brain_trauma/mild/hallucinations/on_life()
+	owner.hallucination = min(owner.hallucination + 10, 50)
+	..()
+
+/datum/brain_trauma/mild/hallucinations/on_lose()
+	owner.hallucination = 0
+	..()
+
+/datum/brain_trauma/mild/stuttering
+	name = "Stuttering"
+	desc = "Patient can't speak properly."
+	scan_desc = "reduced mouth coordination"
+	gain_text = "<span class='warning'>Speaking clearly is getting harder.</span>"
+	lose_text = "<span class='notice'>You feel in control of your speech.</span>"
+
+/datum/brain_trauma/mild/stuttering/on_life()
+	owner.stuttering = min(owner.stuttering + 5, 25)
+	..()
+
+/datum/brain_trauma/mild/stuttering/on_lose()
+	owner.stuttering = 0
+	..()
+#define BRAIN_DAMAGE_FILE "brain_damage_lines.json"
+/datum/brain_trauma/mild/dumbness
+	name = "Dumbness"
+	desc = "Patient has reduced brain activity, making them less intelligent."
+	scan_desc = "reduced brain activity"
+	gain_text = "<span class='warning'>You feel dumber.</span>"
+	lose_text = "<span class='notice'>You feel smart again.</span>"
+
+/datum/brain_trauma/mild/dumbness/on_gain()
+	owner.disabilities |= DUMB
+	..()
+
+/datum/brain_trauma/mild/dumbness/on_life()
+	owner.derpspeech = min(owner.derpspeech + 5, 25)
+	if(prob(3))
+		owner.emote("drool")
+	else if(owner.stat == CONSCIOUS && prob(3))
+		owner.say(pick_list_replacements(BRAIN_DAMAGE_FILE, "brain_damage"))
+	..()
+
+/datum/brain_trauma/mild/dumbness/on_lose()
+	owner.disabilities &= ~DUMB
+	owner.derpspeech = 0
+	..()
+
+#undef BRAIN_DAMAGE_FILE
+
+/datum/brain_trauma/mild/speech_impediment
+	name = "Speech Impediment"
+	desc = "Patient is unable to form coherent sentences."
+	scan_desc = "communication disorder"
+	gain_text = "" //mutation will handle the text
+	lose_text = ""
+
+/datum/brain_trauma/mild/speech_impediment/on_gain()
+	owner.dna.add_mutation(UNINTELLIGIBLE)
+	..()
+
+//no fiddling with genetics to get out of this one
+/datum/brain_trauma/mild/speech_impediment/on_life()
+	if(!(GLOB.mutations_list[UNINTELLIGIBLE] in owner.dna.mutations))
+		on_gain()
+	..()
+
+/datum/brain_trauma/mild/speech_impediment/on_lose()
+	owner.dna.remove_mutation(UNINTELLIGIBLE)
+	..()
+
+/datum/brain_trauma/mild/concussion
```

Comment:
We could probably make this occur to heavy knocks to the head instead of the instant stun.